components/sup/src/config.rs

```diff
@@ -33,12 +33,12 @@ use std::{fmt,
           result,
           str::FromStr};
 
-use crate::{common::{cli_defaults::{GOSSIP_DEFAULT_IP,
-                                    GOSSIP_DEFAULT_PORT,
-                                    GOSSIP_LISTEN_ADDRESS_ENVVAR},
-                     types::EnvConfig},
+use crate::{common::cli_defaults::{GOSSIP_DEFAULT_IP,
```

Comment:
I'm also a fan of having one continuous block of `use` statements at the top of a file (as opposed to having a newline between `std` and everything else. It generally means that `std` is further down the list, but everything is treated consistently... you get `self`, `super`, `crate`, and then everything else in alphabetical order.

Reply:
Yeah I have a pretty strong preference for this style as well.  No more wondering "which block of use statements does this go in", etc.  I'm slowly converting every file I touch to this style.